packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -28,12 +28,10 @@ import { isUndefined, extend } from '../common/utils'
 import Subscription from '../common/subscription'
 import { captureHardNavigation } from './capture-hard-navigation'
 
+const __DEV__ = process.env.NODE_ENV !== 'production'
```

Comment:
Can we use `__DEV__ ` everywhere, instead of using `process.env.NODE_ENV`

Reply:
We cant do that, it can be imported from constant file but wont be dead code eliminated and replaced. so only way is to use process.env in file level.